main/goaccess/APKBUILD

```diff
@@ -28,7 +28,7 @@ build() {
 		--mandir=/usr/share/man \
 		--infodir=/usr/share/info \
 		--localstatedir=/var \
-		--enable-geoip \
+		--enable-geoip=legacy \
```

Comment:
@jirutka Right now the options are `legacy` which will utilize the original GeoIP databases and `mmdb` which will utilize the enhanced GeoIP2 databases.